.tekton/clowder-pull-request.yaml

```diff
@@ -184,7 +189,7 @@ spec:
         - name: name
           value: prefetch-dependencies-oci-ta
         - name: bundle
-          value: quay.io/konflux-ci/tekton-catalog/task-prefetch-dependencies-oci-ta:0.1@sha256:3c11f5de6a0281bf93857f0c85bbbdfeda4cc118337da273fef0c138bda5eebb
+          value: quay.io/konflux-ci/tekton-catalog/task-prefetch-dependencies-oci-ta:0.1@sha256:621b13ab4a01a366a2b1d8403cf06b2b7418afd926d13678c4432858514407d3
```

Comment:
same as before, this is older than upstream, please don't apply this change

https://quay.io/repository/konflux-ci/tekton-catalog/task-prefetch-dependencies-oci-ta?tab=tags&tag=0.1